### PR TITLE
GW-22: Develop about section

### PR DESCRIPTION
**Pull request by chrishadrian**
Developed about section and refactored the folder structure

### CODE REVIEW

**Comment by Julian-UBC** [comment]
@chrishadrian @jenaarianto Here is the website preview: https://deploy-preview-11--gisaubcv3.netlify.app/

**Review by jenaarianto — APPROVED** [review]

**Review by jenaarianto — COMMENTED** [review]
I think it'd be easier on they eyes if the 'who we are' section has a slightly smaller font and di kebawahin more

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-    <link rel="icon" href="%PUBLIC_URL%/gisau-icon.svg" />
+    <link rel="icon" href="%PUBLIC_URL%/gisau_icon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/gisau-icon.svg" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/gisau_icon.svg" />
```
Comment (Julian-UBC) [inline public/index.html:12]: Test comment

File: src/pages/home/components/About.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  const classes = useStyles();
+
+  const handleClickButton = () => {
+    navigate("about");
+  };
+
+  return (
+    <Box className="flex justify-between items-center">
+      <Box className="w-1/2">
+        <Box className={`flex w-2/5 ${classes.bgImage}`}>
```
[the remark this answers is not in the document]
Comment (chrishadrian) [reply src/pages/home/components/About.js]: Fixed

File: src/pages/home/components/LandingImage.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { makeStyles } from "@mui/styles";
 import { CustomButton } from "../../../shared/components/CustomButton";
 import Gisau from "../../../assets/gisau-logo/gisau.png";
-import LandingImg from "../../../assets/landing-image/1.JPG";
+import LandingImg from "../../../assets/landing.jpg";
```
[the remark this answers is not in the document]
Comment (chrishadrian) [reply src/pages/home/components/LandingImage.js:5]: To get better naming and folder structure. What do you think?

File: src/pages/home/components/About.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  button: {
+    borderRadius: "24px",
+    textTransform: "none",
+    width: "22%",
+    background: "linear-gradient(to left, #7D0202, #7D0202)",
+    color: "white",
+    height: "3rem",
+    "&:hover": {
+      background: "linear-gradient(to left, #7D0202, #B25F4C)",
+      transitionTimingFunction: "ease-out",
```
[the remark this answers is not in the document]
Comment (chrishadrian) [reply src/pages/home/components/About.js]: Modified a few properties on the button's styles.

File: src/pages/home/components/LandingImage.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Julian-UBC) [reply src/pages/home/components/LandingImage.js:5]: I had the idea that we will have more than one image for the landing image and not necessarily want to delete the previous one. That's why I put them under a folder.

**Review by Julian-UBC — COMMENTED** [review]
Will just leave two comments here:
1. Somehow the `i` doesn't look good here, the dot seems like merging with the body 
image

2. The button transition still doesn't look smooth enough like the design made by @jenaarianto . This doesn't have to be resolved now. Will just update it in the GW-28.

**Review by Julian-UBC — APPROVED** [review]

**Pull request merged**